projects/addon-editor/components/edit-link/edit-link.component.ts

```diff
@@ -20,18 +26,22 @@ const END = MAX_LENGTH - START - 10;
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TuiEditLinkComponent {
+    @Input()
+    prefix: TuiLinkPrefix = this.makeDefaultPrefix();
+
+    @Input()
+    isOnlyAnchorMode = this.detectAnchorMode();
```

Comment:
Why do we need new inputs? Aren't they all computable from within this component?

Reply:
in tools we need to set static value for indicate that this component only works with anchors
<img width="415" alt="image" src="https://user-images.githubusercontent.com/12021443/197718926-8e9ff822-c733-4e73-b92d-e7ac449f6163.png">

Reply:
And then 
<img width="705" alt="image" src="https://user-images.githubusercontent.com/12021443/197718999-87dc1da3-5f53-4904-8212-fe9620b1e60e.png">

then when the user tries to paste urls there, we will not let him do it

Reply:
fixed